Services/StaticJobProvider.php

```diff
@@ -40,18 +40,22 @@ class StaticJobProvider implements JobProviderInterface
      *
      * @param HostnameManager $hostnameManager
      * @param array           $jobs
+     *
+     * @throws \Exception
```

Comment:
exception isn't throw in this method, so this annotation isn't required

Reply:
done.